core/prepare_api.php

```diff
@@ -94,20 +94,23 @@ function prepare_user_name( $p_user_id ) {
  *
  * @param integer $p_project_id The project id.
  * @param integer $p_version_id The version id.  If false then this method will return an empty string.
+ * @param boolean $p_show_project Force prepending project name to version
```

Comment:
PHPDoc says boolean, but param defaults to _null_. Change PHPDoc type to `boolean|null` and document what null means, or change default param value to _false_ (whatever is appropriate)